extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.quartz.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED, name = "quartz")
+public class QuartzBuildTimeConfig {
+    /**
+     * Enable cluster mode or not.
+     * <p>
+     * If enabled make sure to set the appropriate cluster properties.
+     */
+    @ConfigItem
+    public boolean clustered;
+
+    /**
+     * The type of store to use. Possible values are: `ram`, `db`.
+     * <p>
+     * When using the `db` store type configuration value make sure that you have the datasource configured.
+     * See <a href="https://quarkus.io/guides/datasource-guide"> Configuring your datasource</a> for more information.
+     * <p>
+     * To create Quartz tables, you can perform a schema migration via the <a href="https://quarkus.io/guides/flyway"> Flyway
+     * extension</a> using a `SQL` script matching your database picked from <a href=
```

Comment:
```suggestion
     * extension</a> using a SQL script matching your database picked from <a href=
```
You're using HTML, let's remove the backticks.